sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClientOptions.cs

```diff
@@ -72,6 +78,9 @@ public TextAnalyticsClientOptions(ServiceVersion version = LatestVersion)
         {
             Version = version;
             this.ConfigureLogging();
+
+            //Default Audience to Azure Public Cloud
+            this.Audience = TextAnalyticsAudience.AzureResourceManagerPublicCloud;
```

Comment:
This might be inconsistent with other clients that use `null` as the indicator of default and calculate the actual value later.

Reply:
Other clients throw an exception when null is indicated (https://github.com/Azure/azure-sdk-for-net/blob/53ec34cc1a86c3ffcf9a43811e847e48654bd22a/sdk/containerregistry/Azure.Containers.ContainerRegistry/src/ContainerRegistryClient.cs#L75). I thought if I set the Audience here then if the user does not specify an Audience, we can just default to the Azure Public Cloud without throwing an exception.
@schaabs